experiments/prompt-injection-boundary-tags/rounds/round2b/harness/run_experiment.py

```
@@ -541,28 +576,168 @@ def call_model(model_config: dict, system: str, user_message: str):
+    budget = BudgetController(
+        settings=settings_from_args(args),
+        estimate_cost_usd=estimate_cost_usd,
+        run_id=f"round2b_{timestamp}",
+        timestamp=timestamp,
+        mode="live",
+        planned_trial_models=planned_trial_models,
+        default_report_path=budget_default_report_path,
+    )
 
     print(f"\n{'='*70}")
     print(f"🧪 ROUND 2: Realistic Agent Prompt Injection Defense Experiment")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)